useful_scripts/ros_readbagfile.py

```diff
@@ -132,7 +132,7 @@ def printMsgsInBagFile(args):
         total_count += 1
         no_msgs_found = False
         # Keep track of individual message counters for each message type
-        if msg_counters.has_key(topic) == False:
+        if topic not in msg_counters:
```

Comment:
Ah, I see why this: https://stackoverflow.com/questions/33727149/dict-object-has-no-attribute-has-key/33727186#33727186.